docs/guides/serverless-api-with-planetscale-and-prisma.md

```diff
@@ -0,0 +1,543 @@
+Build and deploy an API with PlanetScale, Prisma and Nitric.
+
+## Overview
+
+This guide shows you how to create an API using serverless functions, then [PlanetScale](https://planetscale.com) and [Prisma](https://www.prisma.io/) for data persistence.
+
+In the guide we'll use:
+
+- [Nitric](https://nitric.io) APIs, Functions and Storage
+- [PlanetScale](https://planetscale.com) serverless database
+- [Prisma](https://prisma.io) ORM
+- The cloud of your choice:
+  - [AWS](https://aws.amazon.com)
+  - [GCP](https://cloud.google.com)
+  - [Azure](https://azure.microsoft.com)
+
+To keep things light, the API we'll create generates memes by allowing uploads of image templates, then generating new images with overlaid text. Feel free to adapt the steps for any other API you'd like to build.
+
+> For image editing, we used a library from NPM called [jimp](https://www.npmjs.com/package/jimp), but you could use anything else you like.
+
+When you're done, you'll have an API which accepts a template image (like the left image below) and can generate multiple memes from it (like the right image below):
+
+<!-- TODO: include image ![meme example]() -->
```

Comment:
Should we remove the image ref for now? Or did you want to add one?

Reply:
Thanks, I removed this one and fixed another image while I was at it